svc-managers/managers/managers.go

```diff
@@ -154,7 +154,7 @@ func (e *ExternalInterface) GetManagers(req *managersproto.ManagerRequest) respo
 					return resp
 				}
 			}
-			err = e.DB.UpdateManagersData(req.URL, managerData)
+			err = e.DB.UpdateData(req.URL, managerData, "Managers")
```

Comment:
What other data can we update from Managers module ?

Reply:
Now same function is used for updating virtualmedia data to DB